File: basic-algorithms/js/02-factorialize-number.js
function factorialize(num, bnoden) {
  bnoden = 1
  for(var i = 1; i <= num; i++) bnoden*=i
  return bnoden
}

// Test
factorialize(5)
factorialize(10)
factorialize(20)
factorialize(0)

File: basic-algorithms/02factorializenumber.go
//https://play.golang.org/p/ffJ2BL_NrQ
package main

import (
	"fmt"
)

func factorialize(num int64) int64 {
	var bnoden, factor int64 = 1, 1

	for ; factor <= num; factor++ {
		bnoden *= factor
	}
	return bnoden
}

// Test
func main() {
	fmt.Println(factorialize(5))  // 120
	fmt.Println(factorialize(10)) // 3628800
	// This is wrong if you use int
	fmt.Println(factorialize(20)) // 2432902008176640000
	fmt.Println(factorialize(0))  // 1
}
